app/viewers/embed/viewer/document_viewer.rb

```diff
@@ -29,13 +29,6 @@ def fullscreen?
         true
       end
 
-      # This indicates if the first PDF is downloadable (though it could be stanford only)
-      # as well as location restricted.  Authorization for other documents will be
-      # checked as the user clicks on items in the content side bar.
-      def available?
-        !document_resource_files.first&.no_download?
-      end
-
       private
 
       def document_resource_files
```

Comment:
I think this method can go too.

Reply:
I think so too, which means we remove pdf_files. I can't quite figure out how the sidebar is getting rendered

Reply:
Ah. I see it is looking at the purl's file.